src/modules/file-explorer/actions/file-explorer.js

```diff
@@ -48,13 +47,33 @@ export default {
 			});
 	},
 
-	delete: (file) => {
-		const currentUser = Server.get('user');
-		return FileService.deleteFile(`users/${currentUser._id}`, file.name, null)
+	delete: (file, storageContext) => {
+		return FileService.deleteFile(storageContext, file.name, null)
 			.then((res) => {
 				return res;
 			}).catch(err => {
 				Notification.showError(err.message);
 			});
+	},
+
+	createNewFolder: (dirName, storageContext) => {
+
+		// load empty fake file for creating new directory
+
+		var options = {
+			responseType: 'blob'
+		};
+
+		return axios.get("/images/.scfake.png", options).then(res => {
+			let fakeFile = res.data;
+			return FileService.getUrl(".scfake.png", fakeFile.type, `${storageContext}/${dirName}`, 'putObject')
+			 .then((signedUrl) => {
+				var options = {
+					headers: signedUrl.header
+				};
+
+				return axios.put(signedUrl.url, fakeFile, options);
+			}).catch(err => Notification.showError(err));
+		});
```

Comment:
This is an AWS-specific hack (to create a folder by uploading a random file). We shouldn't have this in the client.